docs/OpenSSLSupport.md

```diff
@@ -148,6 +148,31 @@ Note that the code snippet for the RDRAND engine opt-in is required to use not o
 but also other OpenSSL APIs that internally depend on the RAND APIs. Alternatively, developers
 can implement their own RAND method to replace the default method via `RAND_set_rand_method` API.
 
+## Security Guidance of using OpenSSL APIs/Macros
```

Comment:
```suggestion
## Security Guidance for using OpenSSL APIs/Macros
```
Note: assuming you accept this suggestion, you will need to update the cross-reference links to this section below.